src/features/api.js

```diff
@@ -0,0 +1,26 @@
+const API_KEY = "9c9166f715ba46b6a9dbd9165c06ab10";
+
+export const getTopHeadlines = async (country, sources = null) => {
```

Comment:
With async function  you need to expect errors, so try to catch all errors by checking the response it is okay